internal/ingress/controller/parser/parser.go

```diff
@@ -573,8 +603,14 @@ func overrideRoute(route *Route,
 	if len(r.Headers) != 0 {
 		route.Headers = r.Headers
 	}
+	// grpc(s) doesn't accept strip_path
 	if len(r.Protocols) != 0 {
 		route.Protocols = cloneStringPointerSlice(r.Protocols...)
+		for _, val := range r.Protocols {
+			if *val == *kong.String("grpc") || *val == *kong.String("grpcs") {
```

Comment:
Why not just `*val == "grpc" || *val == "grpcs`?
`kong.String() is doing nothing but returning a pointer to the string, which you're de-referencing immediately.

Reply:
👍